apps/taiga-lumbermill/src/dashboards/crypto/components/nft/nft-item/nft-item.component.less

```diff
@@ -0,0 +1,56 @@
+.nft-icon {
+    border-radius: 1rem;
+    width: 10rem;
+    height: 10rem;
+}
+
+.info-open {
+    position: relative;
+    display: flex;
+    padding: 1rem;
+    justify-content: space-between;
+}
+
+.close {
+    position: absolute;
+    top: 1rem;
+    right: 0;
+    z-index: 2;
```

Comment:
why do we need z-index?
we should avoid using z-index, it can cause errors and confusion in the future (it also called Z-war) https://www.codegram.com/blog/stop-ab-using-z-index/